File: chessgame-api/src/main/java/git/dimitrikvirik/chessgameapi/service/GameService.java
package git.dimitrikvirik.chessgameapi.service;



import git.dimitrikvirik.chessgameapi.model.domain.Game;
import git.dimitrikvirik.chessgameapi.repository.GameRepository;
import lombok.RequiredArgsConstructor;
import org.springframework.stereotype.Service;

@Service
@RequiredArgsConstructor
public class GameService {
    private final GameRepository gameRepository;



}

File: chessgame-api/src/main/kotlin/git/dimitrikvirik/chessgameapi/service/GameService.kt
package git.dimitrikvirik.chessgameapi.service

import git.dimitrikvirik.chessgameapi.model.redis.Game
import git.dimitrikvirik.chessgameapi.repository.redis.GameRepository
import org.springframework.stereotype.Service
import java.util.*

@Service
class GameService(
    val gameRepository: GameRepository
) {
    fun create(): Game {
        val game = Game(UUID.randomUUID().toString())
        return gameRepository.save(game)
    }
}